assets/state-dra-driver/0200_clusterrole.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nvidia-dra-driver
+  namespace: "FILLED BY THE OPERATOR"
+rules:
+  - apiGroups: ["resource.nvidia.com"]
+    resources: ["computedomains"]
```

Comment:
Why are these permissions listed individually?

Reply:
Since they all have the same verbs, I would lean towards condensing this clusterrole yaml

Reply:
I have condensed the ClusterRole object as suggested. I originally copied the ClusterRole unmodified from the standalone helm chart.